lms/static/scripts/frontend_apps/components/dashboard/test/DashboardActivityFilters-test.js

```diff
@@ -170,11 +175,20 @@ describe('DashboardActivityFilters', () => {
     });
   }
 
+  function createContainerInDOM() {
+    const container = document.createElement('div');
+    containers.push(container);
+    document.body.appendChild(container);
```

Comment:
This function has side effects which make it non-pure, but since we need this in two places, it's useful not having to duplicate the logic.